README-fi.md

```diff
@@ -0,0 +1,24 @@
+# 6Dash
+API ja optimointi modi [3Dash](https://delugedrop.itch.io/3dash):ille
+
+[English](README.md)
+
+## Ominaisuudet
+- Gizmos ([popcron/gizmos](https://github.com/popcron/gizmos) kautta)
+- Kenttä lautaus optimointi
+- Kääntyvät palikat
+- Muita ominaisuuksia jotka voivat auttaa 3Dash modaajia
+
+## Asennus
+1. Lataa BepInEx asennusohjelma [script](https://gist.githubusercontent.com/cgytrus/29085a6bf179893666316a36e1c92bf6/raw/bepinex-installer.ps1)
+2. Laita asennus-scripti sinun 3Dash kansioon
+3. Käynnistä scripti oikea-klikkaamalla ja valitsemalla: `Run with Powershell`
```

Comment:
`Run with Powershell` isn't translated (is it not translated on Windows in Finnish either?)